src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", "Avenir-Heavy", sans-serif;
+  font-weight: 300;
+}
+
+/* html fonts */
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.firstFormInput {
+  width: 100%;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 62px;
+}
+
+.firstFormInput::placeholder {
+  font-family: "Avenir", "Avenir-Heavy", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.firstFormInput:focus {
+  outline: none;
+}
+
+.secondFormInput {
+  width: 100%;
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.secondFormInput::placeholder {
+  font-family: "Avenir", "Avenir-Heavy", sans-serif;
+  font-size: 14px;
+  line-height: 19px;
+  font-weight: 300;
+}
```

Comment:
You duplicate a lot of styles. [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2-nd point